### PR TITLE
Add loved hated view

### PR DESCRIPTION
**Pull request by mayhem**
This is the first step in my project to focus on music discovery workflow -- we really need to be able to see user feedback and manage it. And to listen to it!

image
image

### CODE REVIEW

**Comment by MonkeyDo** [comment]
Could do with more UI tests, but I think we should be ready to go (and I'm done with testing for now) !

File: listenbrainz/db/feedback.py [hunk 1]
```diff
@@ -81,26 +87,71 @@ def get_feedback_for_user(user_id: int, limit: int, offset: int, score: int = No
...
+
+        with timescale.engine.connect() as connection:
+            result = connection.execute(
+                sqlalchemy.text(query), msids=tuple(msids))
+            for row in result.fetchall():
+                if row["recording_mbid"] is not None:
+                    index[row["recording_msid"]].track_metadata['additional_info'] = {
+                        "recording_mbid": row["recording_mbid"],
+                        "release_mbid": row["release_mbid"],
+                        "artist_msid": rec["ids"]["artist_msid"],
```
[the remark this answers is not in the document]
Comment (amCap1712) [reply listenbrainz/db/feedback.py:127]: Also, maybe add a todo or open a ticket to update this code to fetch artist credit mbids as well when listen mapping table is updated with it?

**Review by amCap1712 — APPROVED** [review]
Awesome Work! LGTM

File: listenbrainz/db/feedback.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mayhem) [reply listenbrainz/db/feedback.py:127]: Looks legit to me. Which part do you think is a typo?

**Pull request merged**